render/src/shape_utils.rs

```diff
@@ -676,10 +682,31 @@ pub fn draw_command_fill_hit_test(commands: &[DrawCommand], test_point: swf::Poi
                 winding += winding_number_line(test_point, cursor, *line_to);
                 cursor = *line_to;
             }
-            DrawCommand::CurveTo { control, anchor } => {
+            DrawCommand::QuadraticCurveTo { control, anchor } => {
                 winding += winding_number_curve(test_point, cursor, *control, *anchor);
                 cursor = *anchor;
             }
+            DrawCommand::CubicCurveTo {
+                control_a,
+                control_b,
+                anchor,
+            } => {
+                lyon_geom::CubicBezierSegment {
+                    from: lyon_geom::Point::new(cursor.x.to_pixels(), cursor.y.to_pixels()),
+                    ctrl1: lyon_geom::Point::new(control_a.x.to_pixels(), control_a.y.to_pixels()),
+                    ctrl2: lyon_geom::Point::new(control_b.x.to_pixels(), control_b.y.to_pixels()),
+                    to: lyon_geom::Point::new(anchor.x.to_pixels(), anchor.y.to_pixels()),
+                }
+                .for_each_quadratic_bezier(0.01, &mut |quadratic_curve| {
```

Comment:
Nit: the constant `0.01` is used twice, so it should probably be given a name.

Reply:
Done!